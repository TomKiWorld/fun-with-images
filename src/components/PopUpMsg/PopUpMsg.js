import React from 'react';

const PopUpMsg = ({ message, closePopUp }) => {
  setTimeout(() => {
    const closeBtn = document.querySelector('.close-pop-up')
    if (closeBtn) {
      closeBtn.click();
    }
  }, 8000);

  return (
    <div className='pop-up-message'>
      <span 
        className='close-pop-up' 
        onClick={closePopUp}>&times;</span>
      <p>{message}</p>
    </div>
  );
}

export default PopUpMsg;
