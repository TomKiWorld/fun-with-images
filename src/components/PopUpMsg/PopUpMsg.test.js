import { shallow } from 'enzyme';
import React from 'react';
import PopUpMsg from './PopUpMsg';

describe('PopUpMsg Component', () => {
  const wrapper = shallow(<PopUpMsg message={'Hello'} />);

  it('Expect to render PopUpMsg Component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
