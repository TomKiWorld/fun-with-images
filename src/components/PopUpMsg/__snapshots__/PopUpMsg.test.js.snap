// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PopUpMsg Component Expect to render PopUpMsg Component 1`] = `ShallowWrapper {}`;
